bluetooth/src/commonMain/kotlin/Bluetooth.kt

```diff
@@ -96,6 +97,20 @@ class Bluetooth internal constructor(
 
     private val scanMode = MutableStateFlow<ScanMode>(ScanMode.Stopped)
 
+    override fun pairedDevices(filter: Set<UUID>): Flow<List<Identifier>> = scanningStateRepo
+        .mapLatest { state ->
+            when (state) {
+                is ScanningState.Initialized.Enabled -> state.pairedDevices(filter)
```

Comment:
This paired devices has to live outside of the scanner (it's not related to scanned / discovered list)